java/java.lsp.server/vscode/package.json

```diff
@@ -108,6 +108,34 @@
 					"type": "integer",
 					"default": 100,
 					"description": "Timeout (in milliseconds) for loading Javadoc in code completion (-1 for unlimited)"
+				},
+				"netbeans.java.onSave.organizeImports": {
+					"type": "boolean",
+					"default": true,
+					"description": "Enable organize imports action on a document save"
+				},
+				"netbeans.java.imports.groups": {
+					"type": "array",
+					"description": "Groups of import statements (specified by their package prefixes) and their sorting order. Import statements within a group are ordered alphabetically",
+					"default": [
+						"java",
+						"javax",
+						"org",
+						"com",
+						""
+					]
+				},
+				"netbeans.java.imports.countForUsingStarImport": {
```

Comment:
Do we need an option for wildcard imports at all? It is a bad practice to `import ....*` and I'd just not advertise it in settings at all.

Reply:
Options exported to be on par with RH ext.